DEVDOCS-2178: Update Global Styles (#681)
bigcommerce/dev-docs: docs/stencil-docs/configure-store-design-ui/defining-global-styles.md

* It must be named `config.json`, must reside in the root of your `<theme-name>` top level subdirectory (e.g.: `/cornerstone/config.json`), and must contain valid JSON.
* The maximum allowable size for a theme's `config.json` file is 64 KB. Exceeding this limit will trigger an error upon uploading the theme to BigCommerce.
* Each key's value is restricted to 64 characters. Exceeding this limit will similarly trigger an upload error.

Apart from the aforementioned size constraints, there is no limit on the number of keys and values that you can place in a theme's `config.json`.

<div class="HubBlock--callout">
<div class="CalloutBlock--warning">
<div class="HubBlock-content">

<!-- theme: warning -->

> > Carefully check your theme against all the listed requirements – including the required keys within the `meta` object and `variations` array. While some requirements are not enforced in local development, they will be validated when you upload your theme to BigCommerce.

Author: I think the meaning changes if we say "allowed" instead of "not enforced".

## New products example

> The values that you define in the `config.json` file interact with local resources making it possible to customize your theme's appearance at a global level. Your `config.json` definitions set global defaults for templates, front matter attributes, and Handlebars resources throughout your theme. You can also define custom variables in `config.json`, named according to your needs.

Reviewer: ```suggestion
The values that you define in the `config.json` file interact with local resources making it possible to customize your theme's appearance globally. Your `config.json` definitions set global defaults for templates, front matter attributes, and Handlebars resources throughout your theme. You can also define custom variables in `config.json`, named according to your needs.

```
